src/libraries/System.Security.Cryptography.X509Certificates/tests/DynamicChainTests.cs

```diff
@@ -266,17 +279,9 @@ public static void BasicConstraints_ViolatesCaFalse()
             using (endEntityCert)
             using (intermediateCert)
             using (rootCert)
-            using (ChainHolder chainHolder = new ChainHolder())
             {
-                X509Chain chain = chainHolder.Chain;
-                chain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
-                chain.ChainPolicy.VerificationTime = endEntityCert.NotBefore.AddSeconds(1);
-                chain.ChainPolicy.TrustMode = X509ChainTrustMode.CustomRootTrust;
-                chain.ChainPolicy.CustomTrustStore.Add(rootCert);
-                chain.ChainPolicy.ExtraStore.Add(intermediateCert);
-
-                Assert.False(chain.Build(endEntityCert));
-                Assert.Equal(X509ChainStatusFlags.InvalidBasicConstraints, chain.AllStatusFlags());
+                TestChain3(rootCert, intermediateCert, endEntityCert,
+                    expectedFlags: X509ChainStatusFlags.InvalidBasicConstraints);
```

Comment:
Some formatting change requests:

(Is GitHub wider by default now? This might be a one-liner)

```suggestion
                TestChain3(rootCert, intermediateCert, endEntityCert, expectedFlags: X509ChainStatusFlags.InvalidBasicConstraints);
```

or 

```suggestion
                TestChain3(
                    rootCert,
                    intermediateCert,
                    endEntityCert,
                    expectedFlags: X509ChainStatusFlags.InvalidBasicConstraints);
```

Reply:
> This might be a one-liner

Something I've wondered but failed to find anywhere.. Is there a recommended / documented line length? I did this PR from an iPad so my notion of "width" is probably different. Would be helpful if I could configure a ruler line in VS Code.

Reply:
> Is there a recommended / documented line length?

Not really.  My personal soft limit is at 109 characters (I still cling to a 4:3 monitor for coding, and that's when a horizontal scroll bar pops up for me).  My personal "harder" limit is "there shouldn't be a horizontal scroll bar when viewing the PR from a browser with GitHub's default style".  To say that I have an absolute limit would be a lie.

Reply:
Oh, and while the HSB appears once a line exceeds 109, that's just because VS is too eager to draw an HSB. Things up to 135 characters fit in my aperture without actually requiring scrolling.

So this line as a one-liner wouldn't technically require me to scroll.  If it appeared several times adjacently I'd pull it up to one, but as the last line in the test I'd probably just chop it.

Reply:
Problem solved.

```json
{
    "folders": [
        {
            "path": "./runtime"
        }
    ],
    "settings": {
        "[csharp]": {
            "editor.rulers": [
                109 // @bartonjs danger zone
            ]
        }
    }
}
```